Wireframe/index.html

```diff
@@ -7,27 +7,46 @@
     <link rel="stylesheet" href="style.css" />
   </head>
   <body>
-    <header>
-      <h1>Wireframe</h1>
-      <p>
-        This is the default, provided code and no changes have been made yet.
-      </p>
-    </header>
+  <header>
+  <h1>Understanding Web Concepts</h1>
+  <p>Your go-to guide for foundational web development concepts.</p>
+</header>
+
     <main>
-      <article>
-        <img src="placeholder.svg" alt="" />
-        <h2>Title</h2>
+      <article id='readme'>
+        <img src="placeholder.svg" alt="https://tse3.mm.bing.net/th?id=OIP.6zKkmqpP1a9BLPO1gCV3sQHaFA&pid=Api&P=0&h=180" />
+        <h2>Readme</h2>
+        <p>
+          What is the purpose of a README file?
+         A README file is typically the first document users and developers see when exploring a project.<br>
+          It serves as a guide to understanding the project’s purpose.
+        </p>
+        <a href="https://www.geeksforgeeks.org/what-is-readme-md-file/">Read more</a>
+      </article>
+      <article id='wireframe'>
+        <img  id ='wireframeimg' src='https://cdn-proxy.slickplan.com/wp-content/uploads/2019/08/What-is-Wireframing-768x384.jpg'alt="image for Wireframe sketch" />
+        <h2>Wireframe?</h2>
+        <p>
+         <strong>What is a Wireframe?</strong> A wireframe is a visual blueprint used in UX/UI design to outline the structure of a webpage or application.
+        </p> 
+        <a href="https://www.productplan.com/glossary/wireframe/">Read more</a>
+      </article>
+      <article id='branch'>
+        <img src="placeholder.svg" alt="https://tse3.mm.bing.net/th?id=OIP.BrTvMBJhY4OtenUx5e6Y8gHaDh&pid=Api&P=0&h=180" />
+        <h2> A Branch in a git</h2>
         <p>
-          Lorem ipsum dolor sit amet consectetur adipisicing elit. Quisquam,
-          voluptates. Quisquam, voluptates.
+         <strong>What is a branch in a git?</strong> 
+      A branch in Git is a separate version of your codebase that allows you to develop features, fix bugs, or <br> 
+          experiment without affecting the main code.
         </p>
-        <a href="">Read more</a>
+        <a href="https://www.w3schools.com/git/git_branch.asp">Read more</a>
       </article>
     </main>
     <footer>
       <p>
-        This is the default, provided code and no changes have been made yet.
-      </p>
+<footer>
+  <p> MAY 2025 ITP . Saleh Yousef All rights reserved.</p>
+</footer>      </p>
```

Comment:
We generally try to write code with indentation so it's easy to follow. It's quite hard to know which `<p>` tag this `</p>` tag is closing. Can you reformat this code so it's easier to see?